### PR TITLE
hubble: add livecheck and autobump

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by iMichka** [comment]
Not the latest release https://github.com/cilium/hubble/releases

**Comment by chenrui333** [comment]
- https://github.com/cilium/hubble/issues/1410

File: Formula/h/hubble.rb [hunk 1]
```diff
@@ -1,10 +1,18 @@
 class Hubble < Formula
   desc "Network, Service & Security Observability for Kubernetes using eBPF"
   homepage "https://github.com/cilium/hubble"
-  url "https://github.com/cilium/hubble/archive/refs/tags/v0.13.0.tar.gz"
-  sha256 "c4c2e5ed223c5c4d8777f1da9b44623373240d940365910adb501636a07faecb"
+  url "https://github.com/cilium/hubble/archive/refs/tags/v0.13.1.tar.gz"
+  sha256 "21d7a1e17473b308d3772cf8dd8bec17a15439dce54aa3634a2c4c908a4c04ec"
```
Comment (Bo98) [inline Formula/h/hubble.rb:5]: Did you mean to leave this in

Comment (chenrui333) [reply Formula/h/hubble.rb:5]: yeah, this should be reverted, sorry, I thought I did it. 👍

**Review by Bo98 — APPROVED** [review]

File: Formula/h/hubble.rb [hunk 2]
```diff
@@ -5,6 +5,14 @@ class Hubble < Formula
   sha256 "c4c2e5ed223c5c4d8777f1da9b44623373240d940365910adb501636a07faecb"
   license "Apache-2.0"
 
+  # Upstream creates stable version tags (e.g., `v1.2.3`) before a release but
+  # the version isn't considered to be released until a corresponding release
+  # is created on GitHub, so it's necessary to use the `GithubLatest` strategy.
```
Comment (samford) [inline Formula/h/hubble.rb:10]:
```suggestion
  # There can be a notable gap between when a version is tagged and a
  # corresponding release is created, so we check the "latest" release instead
  # of the Git tags.
```

The "isn't considered to be released until a corresponding release is created on GitHub" comment is for projects that explicitly use GitHub releases to indicate that a version is released (i.e., confirmed by upstream). Upstream hasn't commented on the related issue yet, so the "notable gap" comment is arguably more appropriate here (it's more of an indication of how we handle version releases in homebrew/core).

**Review by samford — APPROVED** [review]

**Pull request merged**